client/components/ReusableErrorList.js

```diff
@@ -0,0 +1,27 @@
+const humanizeErrors = errors => {
+  errorContainer = errors.map(error => {
+    switch (error) {
+      case "phoneNumber":
+        error = "phone number";
+        break;
+      case "petType":
+        error = "pet type";
+        break;
+      case "petAge":
+        error = "pet age";
+        break;
+      case "petImage":
+        error = "pet image";
+        break;
+      case "petName":
+        error = "pet name";
+        break;
+      case "vaccinationStatus":
+        error = "vaccination status";
+        break;
+    }
+    return <p>{`${_.capitalize(error)} is empty or not selected.`}</p>
+  })
+}
```

Comment:
Not a change that needs to be made, but they have library's to help with cameCase converting
https://humps.readthedocs.io/en/latest/